.envrc

```diff
@@ -0,0 +1,8 @@
+use_flake() {
+  watch_file flake.nix
+  watch_file flake.lock
+  eval "$(nix print-dev-env --profile "$(direnv_layout_dir)/flake-profile")"
+}
+echo "here"
+mkdir -p .direnv
+use flake
```

Comment:
This and the flake files are useful only if you're running NixOS or the Nix package manager. They're safe to ignore otherwise.